docs/dev/contributing.rst

```diff
@@ -230,9 +230,11 @@ be aware of the following things when filing bug reports:
 Run the tests
 -------------
 
-Two ways of running the tests are as follows:
+There are a few ways of running the tests:
 
-1. ``./run-tests.sh`` or ``run-tests.bat``
+1. run-tests.sh
```

Comment:
Might be worth noting how the CI runs the tests -- that is the file I always reference for running them.
One time setup:
```
git submodule sync
git submodule update --init --recursive
python -m pip install -e . --upgrade
pipenv install --deploy --dev
```

Run tests: `pipenv run pytest -ra -n auto -v --fulltrace tests`

Reply:
Nevermind, it is right below this already:  https://github.com/pypa/pipenv/pull/5182/files#diff-b423d06524f8dec680323ad0294641ed796dfa5b2c876be7d84df83d21d374f5R264-R268